### PR TITLE
fix: type definitions in NIPALS

### PR DESCRIPTION
**Pull request by josoriom**

### CODE REVIEW

**Comment by targos** [comment]
there's something wrong with the Prettier config.

File: package.json [hunk 1]
```diff
@@ -16,10 +16,12 @@
...
+    "eslint": "eslint src testUtils.js",
     "eslint-fix": "npm run eslint -- --fix",
-    "prepublishOnly": "npm run compile",
-    "test": "npm run test-coverage && npm run eslint",
+    "prepack": "npm run compile",
+    "prettier": "prettier --check src",
+    "prettier-write": "prettier --write src",
+    "test": "npm run test-coverage && npm run eslint && npm run prettier",
     "test-only": "jest",
     "test-coverage": "jest --coverage"
```
Comment (targos) [inline package.json:26]:
```suggestion
    "test": "npm run test-only && npm run eslint && npm run prettier",
    "test-only": "jest --coverage"
```

**Comment by targos** [comment]
I think the problem is that there is no Prettier config at all in this project, so Prettier uses its defaults.
Try to add a prettierrc like this one: https://github.com/mljs/spectra-processing/blob/master/.prettierrc.json

**Comment by josoriom** [comment]
@targos Could this error be related to the nodejs.yml file?

File: .github/workflows/typedoc.yml [hunk 2]
```diff
@@ -1,22 +1,32 @@
 name: Deploy TypeDoc on GitHub pages
 
 on:
+  workflow_dispatch:
   release:
     types: [published]
 
+env:
+  NODE_VERSION: 16.x
+  ENTRY_FILE: 'src/index.ts'
```
Comment (targos) [inline .github/workflows/typedoc.yml:10]:
```suggestion
  ENTRY_FILE: 'matrix.d.ts'
```

**Pull request merged**